src/Button.js

```diff
@@ -169,7 +186,11 @@ Button.propTypes = {
    * FAB Click-Only
    * Turns a FAB from a hover-toggle to a click-toggle
    */
-  fabClickOnly: PropTypes.bool
+  fabClickOnly: PropTypes.bool,
+  /**
+   * Styles
+   */
+  style: PropTypes.object
```

Comment:
For some reason styles weren't working (e.g `<Button fab="true" style={{...}}>...</Button>`). The problem only happens when using fab button, so I only adapted renderFab function.

Would it be nice if we create a `<FABButton>` or `<FixedFloatingButton>...<Button>...</Button><FixedFloatingButton>` in the future so we might have better organized component logic?

@alextrastero

Reply:
The reason that `styles` or other dom attributes are not passed is because we specifically use `{...other}` on L79. So I would use a different approach: 

1. `renderFab` should read directly from `this.props` for that we might need to bind it in the constructor.
2. pass `{...other}` to the `div` like we do on L79

Bare in mind though that this PR should only handle one topic, and in this case it's adding the stories :) If we have an implementation bug let's tackle that separately with it's own specs/PR. Can we remove the commits for the `Button` changes and put them in another PR?

Reply:
I see.

Some of the stories (mainly fab ones) need those fixes in order to work. I will do as your advice and put those Button fixes into a different PR.

Thanks @alextrastero

Reply:
I see, have a look here to see how you can add a style decorator to the stories that need it :)
https://github.com/react-materialize/react-materialize/blob/3.0.0-beta.0/stories/Collection.stories.js#L16